src/items/LinkItem.tsx

```diff
@@ -60,12 +46,12 @@ const LinkItem: FC<LinkItemProps> = ({
   saveButtonId,
   editCaption = false,
   showCaption = true,
-  loadingMessage = 'Link is Loading.',
-  openLinkMessage = 'Click here to open link manually.',
+  loadingMessage = 'Link is Loading...',
+  openLinkMessage = 'Click here to open the link manually',
   height: defaultHeight,
+  errorMessage = 'The link is malformed.',
```

Comment:
Are these messages hard-coded? Or can they be overridden when used? It's probably better if everything can be overridden.

Reply:
These are default values, so yes they can be overridden!